helper.php

```diff
@@ -392,6 +397,23 @@ function _descCell() {
     }
 
     /**
+     * Diff icon / link to diff page
+     */
+    function _diffCell($id) {
+        // check for page existence
+        if (!isset($this->page['exists'])) {
+            if (!isset($this->page['file'])) $this->page['file'] = wikiFN($id);
+            $this->page['exists'] = @file_exists($this->page['file']);
+        }
+
+        // produce output
+        $content = '<a href="'.wl($id).($this->page['section'] ? '#'.$this->page['section'] : '').'&do=diff" class="diff_link">
```

Comment:
The anchor needs to be after the get parameters. In order to ensure correct parameter separators please pass the diff parameter to the `wl`-function.